app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardNetworksStep/MappingWizardNetworksStepActions.js

```diff
@@ -5,6 +5,7 @@ import {
   FETCH_V2V_TARGET_NETWORKS,
   QUERY_ATTRIBUTES
 } from './MappingWizardNetworksStepConstants';
+import { V2V_TARGET_PROVIDER_NETWORK_KEYS } from '../../MappingWizardConstants';
```

Comment:
I think you need to add this constant still 😛

Reply:
yep, that's what i get for working off the branch of a branch 🤦‍♂️ ❤️